health-services/health-notification-service/LOCALSETUP.md

```
@@ -0,0 +1,92 @@
+
+1. Start Zookeeper:
+```bash
+bin/zookeeper-server-start.sh config/zookeeper.properties
+```
+
+2. Start Kafka:
+```bash
+bin/kafka-server-start.sh config/server.properties
+```
+
+3. Create topics:
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)